--- origsrc/uni2ascii-4.8/Makefile.am	2008-04-26 12:26:49.000000000 -0500
+++ src/uni2ascii-4.8/Makefile.am	2008-05-05 21:02:20.343750000 -0500
@@ -7,7 +7,7 @@
 AUTOMAKE_OPTIONS = dist-zip dist-bzip2
 EXTRA_DIST = TestSuiteAscii2Uni CREDITS uni2html.py ascii2uni.py u2a.tcl uni2ascii.info
 u2a:	u2a.tcl
-	cp u2a.tcl u2a
+	cp $< $@
 
 
 
--- origsrc/uni2ascii-4.8/configure.ac	2008-05-04 02:14:48.000000000 -0500
+++ src/uni2ascii-4.8/configure.ac	2008-05-05 21:06:14.718750000 -0500
@@ -19,6 +19,7 @@
 AC_FUNC_MALLOC
 AC_FUNC_REALLOC
 AC_CHECK_FUNCS([regcomp setlocale strtoul])
+AC_SEARCH_LIBS(gettext, intl)
 
 AC_CONFIG_FILES([Makefile])
 AC_OUTPUT
